data/net/minecraft/util/Mth.mapping

```diff
@@ -6,10 +6,27 @@ CLASS net/minecraft/util/Mth
 	METHOD abs (I)I
 		COMMENT Returns the unsigned value of an int.
 		ARG 0 value
+	METHOD absFloor (D)I
+		ARG 0 value
 	METHOD absMax (DD)D
 		COMMENT Maximum of the absolute value of two numbers.
 		ARG 0 x
 		ARG 2 y
+	METHOD approach (FFF)F
+		COMMENT Changes value by stepSize towards the limit and returns the result.
+		COMMENT If value is smaller than limit, the result will never be bigger than limit.
+		COMMENT If value is bigger than limit, the result will never be smaller than limit.
+		ARG 0 value
+		ARG 1 limit
+		ARG 2 stepSize
+	METHOD approachDegrees (FFF)F
+		COMMENT {@link #approach(float, float, float)} for rotations.
```

Comment:
The first sentence of the method's javadoc should describe what it does, and a link to another method should be in the succeeding sentences or in a `@see` tag. It should also explain the behavior in which it differs from the regular `approach` (with respect to that call to `degreesDifference`).